carta/session.py

```diff
@@ -369,33 +367,33 @@ def open_complex_image(self, path, expression=ComplexExpression.AMPLITUDE, appen
             The path to the complex-valued image file, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
         expression : {1}
             The complex expression to use when opening the image. The default is :obj:`carta.constants.ComplexExpression.AMPLITUDE`.
-        append : {2}
+        hdu : {2}
+            The HDU to select inside the file.
+        append : {3}
             Whether the image should be appended to existing images. By default this is ``False`` and any existing open images are closed.
-        update_directory : {3}
+        update_directory : {4}
             Whether the starting directory of the frontend file browser should be updated to the parent directory of the image. The default is ``False``.
         """
-        path = self.resolve_file_path(path)
-        directory, file_name = posixpath.split(path)
-        expression = f'{expression}("{file_name}")'
-        return Image.new(self, directory, expression, "", append, True, update_directory=update_directory)
+        return Image.new(self, path, hdu, append, True, True, expression, update_directory=update_directory)
 
-    @validate(String(), String(), Boolean(), Boolean(), Boolean())
-    def open_LEL_image(self, expression, directory=".", append=False, update_directory=False):
+    @validate(String(), String(), String(), Boolean(), Boolean())
+    def open_LEL_image(self, expression, path=".", hdu="", append=False, update_directory=False):
```

Comment:
As discussed above, the way you have changed these parameters and the way you are calling the new function from this function does not make sense and will not produce the correct result.